test/transaction/transaction.js

```diff
@@ -401,6 +401,17 @@ describe('Transaction', function() {
         transaction.addInput(new Transaction.Input(), Script.empty(), 0);
       }).to.not.throw();
     });
+    it('does not allow a threshold number greater than the amount of public keys', function() {
+      expect(function() {
+        transaction = new Transaction();
+        return transaction.from({
+          txId: '0000000000000000000000000000000000000000000000000000000000000000',
+          outputIndex: 0,
+          script: Script.buildMultisigOut([public1, public2], 2).toScriptHashOut(),
```

Comment:
This should be a different test (the call to `from` is expected to fail, not the creation of the multisig)